research/object_detection/exporter.py

```diff
@@ -69,7 +69,7 @@ def freeze_graph_with_def_protos(
     if optimize_graph:
       logging.info('Graph Rewriter optimizations enabled')
       rewrite_options = rewriter_config_pb2.RewriterConfig(
-          optimize_tensor_layout=True)
+          layout_optimizer=1)
```

Comment:
Can you change this to layout_optimizer=rewriter_config_pb2.RewriterConfig.ON? This makes our intentions more explicit.

Reply:
Agreed and made the change. Could you review.